### PR TITLE
Swift support

### PR DESCRIPTION
**Pull request by Ruenzuo**
This PR contains small changes to enable Swift support on XNGAPIClient.

* Fixes wrong/missing import directives: https://github.com/xing/XNGAPIClient/commit/8622b572af2e66031c7fc404cadf2b811b0c2aab https://github.com/xing/XNGAPIClient/commit/1c17b698fdb26878cadecc30112e8eb101300ea1
* Adds `AFNetworking` as a dependency. For some reason, without this Xcode is not able to build the project using `use_frameworks!` on the Podfile. I'll double check this with the CocoaPods team though, I feel I'm missing something obvious here, because this dependency should be resolved by CocoaPods.

### CODE REVIEW

**Comment by Ruenzuo** [comment]
Update on the `use_frameworks!` thingie: https://github.com/CocoaPods/CocoaPods/issues/4141

File: XNGAPIClient/XNGAPIClient.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 // THE SOFTWARE.
 
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
```
Comment (pietbrauer) [inline XNGAPIClient/XNGAPIClient.h]: Why is that needed?

Comment (Ruenzuo) [inline XNGAPIClient/XNGAPIClient.h]: Xcode complains about UIImage references on [some categories](https://github.com/xing/XNGAPIClient/blob/5fa8fc6f27b38603a1db4ce98b83de184db405b2/XNGAPIClient/XNGAPIClient%2BProfileEditing.m#L47-L49). I find it nice putting it here, because it makes clear that XNGAPIClient requires UIKit but I guess we could move it to the specific categories where UIKit classes are referenced.

Comment (pietbrauer) [inline XNGAPIClient/XNGAPIClient.h]: Ah right, I couldn't remember what it was.
